tests/parser/types/numbers/test_sqrt.py

```diff
@@ -0,0 +1,63 @@
+from decimal import (
+    ROUND_FLOOR,
+    Decimal,
+)
+
+
+def decimal_sqrt(val):
+    return val.sqrt().quantize(
+        Decimal('0.0000000000'),
+        rounding=ROUND_FLOOR
+    )
```

Comment:
Just to confirm, this will match up to 10 decimal places, correct?

Reply:
Yes, took me a while to find it ;)

Reply:
Make sure to test around the edge of this a little better (i.e. something at 9, 10, 11 decimal places)

Reply:
Not sure what you mean? The round/truncate is applied after the square root ( which has way more digits than 10).

Reply:
Oh, I mean that the test cases you run through contract produce results that have significant figures at 9, 10, and 11 decimals in length, to verify this truncation does not lose precision.